src/components/list/template.njk

```diff
@@ -17,8 +18,8 @@
 
 {%- for item in params.items %}
   <li>
-    {% if (params.isStep) or (params.isStepLarge) %}
-    <span class="govuk-c-list__icon govuk-u-circle {% if (params.isStepLarge) %}govuk-c-list__icon--large{% endif %}">{{ loop.index }}</span> {{ item.content }}
+    {% if (type == 'step') or (type == 'step--large') %}
```

Comment:
A lot of these brackets seem unnecessary?

Reply:
oh yeah, not really. will remove